src/libraries/System.Reflection/tests/System.Reflection.Tests.csproj

```diff
@@ -69,4 +69,14 @@
     <WasmFilesToIncludeFromPublishDir Include="$(AssemblyName).dll" />
     <WasmFilesToIncludeFromPublishDir Include="$(AssemblyName).pdb" />
   </ItemGroup>
+
+  <Target Name="ExcludeAssembliesFromSingleFile"
```

Comment:
Can this be a common scenario in multiple test assemblies? Should we instead add a property, or itemgroup that lists the files that we want to exlude from single file, and if that property/item is not empty then run this target and probe %(FileName) against that property/item?